plugin.video.plutotv/resources/lib/plutotv.py

```diff
@@ -41,23 +40,27 @@
 USER_EMAIL   = REAL_SETTINGS.getSetting('User_Email')
 PASSWORD     = REAL_SETTINGS.getSetting('User_Password')
 USER_REGION  = REAL_SETTINGS.getSetting("Select_Country")
+HIDE_PLUTO   = True
 FIT_REGION   = False if USER_REGION == 'US' else True
 DEBUG        = REAL_SETTINGS.getSetting('Enable_Debugging') == 'true'
 COOKIE_JAR   = xbmc.translatePath(os.path.join(SETTINGS_LOC, "cookiejar.lwp"))
 PTVL_RUN     = xbmcgui.Window(10000).getProperty('PseudoTVRunning') == 'True'
-HIDE_PLUTO   = True
 IGNORE_KEYS  = ['Pluto TV Device Promo 15s prod','Space Station 10s - Promo','Pluto TV Device Promo 15s prod','Exploding Logo 5s','Pluto TV 5 Minute Spot Promo','vibes promo 5s']
 YTURL        = 'plugin://plugin.video.youtube/play/?video_id='
 VMURL        = 'plugin://plugin.video.vimeo/play/?video_id='
 BASE_URL     = 'http://pluto.tv/'#'http://silo.pluto.tv/'
-BASE_API     = 'https://api.pluto.tv/v1'
-BASE_LINEUP  = 'https://api.pluto.tv/v1/channels.json'
-BASE_GUIDE   = 'https://api.pluto.tv/v1/timelines/%s.000Z/%s.000Z/matrix.json'
-BASE_CLIPS   = 'https://api.pluto.tv/v2/episodes/%s/clips.json'
+BASE_API     = 'https://api.pluto.tv'
+BASE_LINEUP  = BASE_API + '/v1/channels.json'
+BASE_GUIDE   = BASE_API + '/v1/timelines/%s.000Z/%s.000Z/matrix.json'
+LOGIN_URL    = BASE_API + '/v1/auth/local'
+BASE_CLIPS   = BASE_API + '/v2/episodes/%s/clips.json'
 PLUTO_MENU   = [("Browse Channels" , BASE_LINEUP, 0),
                 ("Browse OnDemand" , BASE_LINEUP, 1),
                 ("Channel Guide"   , BASE_LINEUP, 20)]
               
+def isUWP():
+    return len(fnmatch.filter(glob.glob(LANGUAGE(30011)),'*.*') + fnmatch.filter(glob.glob(LANGUAGE(30012)),'*.*')) > 0
```

Comment:
```python
def isUWP():
    sys.platform == "win10"
```

Reply:
Thanks, however "Win10" doesn't include Xbox UWP. Also not all "Win10" users installed through the app store so they may not be running the UWP version of Kodi.

Reply:
`win10` means uwp platform, on PC, Xbox or mobile

Reply:
on native windows 10 it has `win32` value

Reply:
Okay cool... I'll use this method: thanks.